lib/fueling.rb

```diff
@@ -3,11 +3,9 @@
 ActiveRecord.default_timezone = :local
 
 class Fueling < ActiveRecord::Base
+  # :nocov:
   self.table_name_prefix = 'emsdba.' unless ENV['RACK_ENV'] == 'test'
+  # :nocov:
   self.table_name = 'FTK_MAIN'
   self.primary_key = 'row_id'
-
-  def readonly?
```

Comment:
I'd kinda like to keep this; the README recommends using a readonly user, but I _really_ don't want to risk writing to the Trapeze DB from anything other than Trapeze.

As I'm thinking about it, both this and the `table_name_prefix` above belong in the `ApplicationRecord` if #120 is merged first.

Reply:
It totally didn't occur to me this is an AR API definition (thought it was just unused)

Shall write a spec in that case